apps/journeys-admin/src/components/TemplateView/TemplatePreviewTabs/TemplatePreviewTabs.tsx

```diff
@@ -44,17 +64,27 @@ export function TemplatePreviewTabs(): ReactElement {
         sx={{ borderBottom: 1, borderColor: 'divider', mb: { xs: 3, sm: 7 } }}
       >
         <Tab
-          disableRipple
-          label={t('{{cardBlockCount}} Cards', {
-            cardBlockCount: steps?.length ?? 0
+          label={t('{{count}} Cards', {
+            count: steps?.length ?? 0
           })}
           {...tabA11yProps('cards-preview-tab', 0)}
-          sx={{ width: smUp ? 200 : undefined, pointerEvents: 'none' }}
+          sx={{ width: { xs: undefined, sm: 200 } }}
```

Comment:
```suggestion
          sx={{ width: { sm: 200 } }}
```

Why does this need to be set at all?

Reply:
to match design